Makefile

```diff
@@ -45,7 +45,7 @@ endif
 ifeq ($(OS),Darwin)
 	goreleaser build --snapshot --clean --config $(CURDIR)/.goreleaser.macos-latest.yml
 else ifeq ($(OS),Linux)
-	goreleaser build --snapshot --clean --config $(CURDIR)/.goreleaser.ubuntu-20.04.yml
+	goreleaser build --snapshot --clean --config $(CURDIR)/.goreleaser.ubuntu-latest.yml
```

Comment:
`ubuntu-20.04` is not running from `Makefile`?

Reply:
Not in this PR. I considered this and thought a separate build target (`build-u2f`) night be the simplest way to achieve this, but thought of doing that as a separate PR if necessary. Can roll it into this one of you prefer, but not until Tuesday at the earliest.